extensions/packageManager/src/node/index.ts

```diff
@@ -26,6 +27,26 @@ const isAdaptiveComponent = (c) => {
   return hasSchema(c) || c.includesExports;
 };
 
+const loadReadme = async (components) => {
+  const variants = ['readme.md', 'README.md', 'README.MD'];
+  for (const c in components) {
+    if (components[c].path) {
+      const rootFolder = path.dirname(components[c].path);
+      for (const v in variants) {
+        const readmePath = path.join(rootFolder, variants[v]);
+        // eslint-disable-next-line security/detect-non-literal-fs-filename
+        if (fs.existsSync(readmePath)) {
+          // eslint-disable-next-line security/detect-non-literal-fs-filename
+          components[c].readme = fs.readFileSync(readmePath, 'utf-8');
```

Comment:
a readme file can be big, I suggest async read here

Reply:
Ben resolved this in his latest push